SPECS/conda/conda.spec

```diff
@@ -0,0 +1,449 @@
+%{!?_with_bootstrap: %global bootstrap 0}
+
+Summary:        Cross-platform, Python-agnostic binary package manager
+Name:           conda
+Version:        4.10.1
+Release:        2%{?dist}
+License:        BSD and ASL 2.0 and LGPLv2+ and MIT
+# The conda code is BSD
+# progressbar is LGPLv2+
+# six is MIT/X11
+# adapters/ftp.py is ASL 2.0
+URL:            http://conda.pydata.org/docs/
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://github.com/conda/conda/archive/%{version}/%{name}-%{version}.tar.gz
+# bash completion script moved to a separate project
+Source1:        https://raw.githubusercontent.com/tartansandal/conda-bash-completion/1.5/conda
+Patch0:         conda_sys_prefix.patch
+Patch1:         conda_gateways_disk_create.patch
+Patch2:         setup.patch
+# Use system cpuinfo
+Patch3:         conda-cpuinfo.patch
+
+Patch10001:     0001-Fix-toolz-imports.patch
+Patch10003:     0003-Drop-fs-path-encoding-manipulation-under-python2.patch
+Patch10004:     0004-Do-not-try-to-run-usr-bin-python.patch
+Patch10005:     0005-Fix-failing-tests-in-test_api.py.patch
+Patch10006:     0006-shell-assume-shell-plugins-are-in-etc.patch
+
+BuildArch:      noarch
+
+BuildRequires:  bash-completion
+%global bash_completionsdir %(pkg-config --variable=completionsdir bash-completion 2>/dev/null || echo '/etc/bash_completion.d')
+BuildRequires:  sed
+
+Requires:       python%{python3_pkgversion}-conda = %{version}-%{release}
+# Removed upstream in favour of calling "conda activate" in version 4.4.0
+Obsoletes:      conda-activate < 4.4
```

Comment:
We should be avoiding "Obsoletes" for packages we've never shipped, it's just another item taking up space in the package graph